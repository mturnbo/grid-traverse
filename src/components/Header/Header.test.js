import React from 'react';
import { mount } from 'enzyme';
import HeaderComponent from './index';

const title = 'This Is A Test';
const wrapper = mount(<HeaderComponent title={title} />);

test('Header Component Renders', () => {
  expect(wrapper).toMatchSnapshot();
});

test('Header Component Title', () => {
  expect(wrapper.prop('title')).toEqual(title);
});
