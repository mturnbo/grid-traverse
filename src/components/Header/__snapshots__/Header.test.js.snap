// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Renders 1`] = `
<Header
  title="This Is A Test"
>
  <header
    className="app-header"
  >
    <h2>
      This Is A Test
    </h2>
  </header>
</Header>
`;
